portfolio/src/main/webapp/aboutme.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About Claudia</title>
+    <link rel="stylesheet" href="aboutme.css">
+        <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <h2> About me: </h2>
```

Comment:
I think you can remove the whitespace before and after text because they are ignored by HTML. https://google.github.io/styleguide/htmlcssguide.html#Trailing_Whitespace
Here and in other places.